langlib/lang.float/src/main/ballerina/float.bal

```diff
@@ -366,6 +526,12 @@ public isolated function toHexString(float x) returns string = @java:Method {
 # - `NaN` is allowed
 # - `Infinity` is allowed with an optional leading `+` or `-` sign
 #
+# ```
+# float:fromHexString("0x1.0a3d70a3d70a4p4") ⇒ 16.64
+# 
+# float:fromHexString("John") ⇒ error
```

Comment:
Let's use a different example, where the string is more closer to being a valid one.

Reply:
I think we need an example that clearly show the difference,
When user see the example they should get the idea that, other strings will not work in here.

Reply:
The fact that "John" is not a hex string and therefore cannot be used to create a float is obvious, right? Why not something like 

```suggestion
# float:fromHexString("0x1J") ⇒ error
```